staff/carlos-corredor/youtube-channel-template/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Youtube Carlos C</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+</head>
+<body>
+    <header>
+        <div class="header left-h">
+            <a href="https://www.youtube.com/" target="_blank"><img src="https://wh1k8zidop.inscname.net/big/1477488.jpg?v=1545841812" alt="Youtube enlace" width="70px" height="35px"></a>
+            <h5 class="h-header">Boris Brejcha</h5>
+        </div>
+        <div class="header right-h">
+            <a href="#"><img src="http://www.clker.com/cliparts/F/2/r/b/e/n/white-magnifying-glass-hi.png" alt="Buscar" width="15px" height="15px"></a>
+            <button class="button-header"><img src="images/icon-ui-1-options-512w.png" alt="Opciones de Youtube" width="20" height="20"></button>
+        </div>
+        
+    </header>
+    <section class="subh">
+        <div class="isubh">
```

Comment:
@Carlos7979 you have to look at BEM and rename all classes. BEM works as it follows:
A header could be:
`header` (block), `header__title`(block+element), `header__img`(block+element), `header__quote`(block+element), `header__quote-link`(block+element+element), `header__quote-link--used` (block+element+element+modifier). Please rename them using BEM.